acceptance/windows-server-2012r2-checksum/terraform/application.tf

```diff
@@ -0,0 +1,94 @@
+data "aws_ami" "windows_ami" {
+  most_recent = true
+
+  filter {
+    name   = "owner-alias"
+    values = ["amazon"]
+  }
+
+  filter {
+    name   = "name"
+    values = ["Windows_Server-2012-R2*-English-*-Base-*"]
+  }
+
+  filter {
+    name   = "architecture"
+    values = ["x86_64"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  filter {
+    name   = "block-device-mapping.volume-type"
+    values = ["gp2"]
+  }
+
+  filter {
+    name   = "image-type"
+    values = ["machine"]
+  }
+}
+
+resource "aws_instance" "mixlib_install_ps1" {
+  count = 1
+
+  ami           = "${data.aws_ami.windows_ami.id}"
+  instance_type = "${var.aws_instance_type}"
+  key_name      = "es-infrastructure"
+
+  associate_public_ip_address = true
+
+  subnet_id         = "subnet-11ac0174" # Planet Releng Public Subnet
```

Comment:
Is this the account we want to use for this, I am fine if it is -- I just want to make sure though.

Reply:
For now yes. I'm hoping within the next few weeks to convert the acceptance tests over to using expeditor's workflow. Then we'll switch to the es account.